lib/releases.sh

```diff
@@ -40,11 +41,20 @@ else
   export CAPM3RELEASE="${CAPM3RELEASE:-$(get_latest_release "${CAPM3RELEASEPATH}" "v0.3.")}"
 fi
 
+
 # On first iteration, jq might not be installed
 if [[ "$CAPIRELEASE" == "" ]]; then
   command -v jq &> /dev/null && echo "Failed to fetch CAPI release from Github" && exit 1
 fi
 
 if [[ "$CAPM3RELEASE" == "" ]]; then
   command -v jq &> /dev/null && echo "Failed to fetch CAPM3 release from Github" && exit 1
-fi
\ No newline at end of file
+fi
```

Comment:
remove this block. 
you are hard coding the values here. They need to be set somewhere else.